wikimedia/wikipedia-ios
# Image recommendations -  Data transition to preview article

**Phabricator:** https://phabricator.wikimedia.org/T359225

### Notes
* This PR adds a fetch wikitext call to the insert media settings VC when coming from the image recs feature
* It then uses the wikitext utility to compose the new updated wikitext
* No error handling is being done, as there are no checks for user permissions - assuming only unblocked users can access the feature and no locked articles are made available by the API

### Test Steps
1. Go through the image recs feature on Staging
2. Verify the wikitext being printed is correct
3. Article preview is being handled in another ticket
4. Go through the regular add image edit flow, it should work as usual
